### PR TITLE
Remove all cozy* UNIX users when purging.

### LINKED ISSUE #35: Delete cozy-* users when purging
We should find a way to delete every users created by Cozy.

### PR DESCRIPTION
**Pull request by babolivier**
Fix #35

### CODE REVIEW

File: debian/cozy-standalone.postrm [hunk 1]
```diff
@@ -44,6 +44,12 @@ case "$1" in
 		[ -d /usr/local/var/log/cozy ] && echo "Delete /usr/local/var/log/cozy directory" && rm -rf /usr/local/var/log/cozy
 		[ -d /usr/local/cozy ] && echo "Delete /usr/local/cozy directory" && rm -rf /usr/local/cozy
 		[ -d /var/www/cozy-letsencrypt ] && echo "Delete /var/www/cozy-letsencrypt directory" && rm -rf /var/www/cozy-letsencrypt
+		
+		echo "Delete Cozy's UNIX users"
+		users=`cat /etc/passwd | cut -d":" -f1 | grep "^cozy\-"`
+		for user in $users; do
+			userdel $user
```
Comment (nono) [inline debian/cozy-standalone.postrm:51]: @babolivier [the manpage](http://manpages.ubuntu.com/manpages/trusty/man8/userdel.8.html) says:

>  userdel is a low level utility for removing users. On Debian, administrators should usually use deluser(8) instead.

And shouldn't we delete the home directory as well?

**Comment by frankrousseau** [comment]
@nledez Could you have a look at this PR when you have time for it?

File: debian/cozy-standalone.postrm (hunk 1 above)
Comment (babolivier) [inline debian/cozy-standalone.postrm:51]: The homes directory are located in `/usr/local/cozy/apps`, which is already removed [here](https://github.com/babolivier/cozy-debian/blob/054d05fc464bf48d0cfa8efd62a5d0c32779e6fd/debian/cozy-standalone.postrm#L45)

**Comment by nono** [comment]
@nledez ping

**Pull request merged**